### PR TITLE
Adding featuretools_sql

### PR DESCRIPTION
**Pull request by sbadithe**
- Adds `featuretools_sql` to `conda-forge`

### CODE REVIEW

File: recipes/featuretools_sql/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  imports:
+    - featuretools_sql
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/featuretools-sql/
+  summary: directly import and convert their relational data into a Featuretools
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:40]:
```suggestion
  summary: directly import your relational data into Featuretools to run automated feature engineering
```

File: recipes/featuretools_sql/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - alteryx
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:46]:
```suggestion
    - stefaniesmith
    - sbadithe
    - ozzieD
    - machineAYX
    - rwedge
    - tamargrey
    - thehomebrewnerd
    - gsheni
    - dvreed77
```

File: recipes/featuretools_sql/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  imports:
+    - featuretools_sql
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/featuretools-sql/
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:39]:
```suggestion
  doc_url: https://featuretools.alteryx.com/en/stable/guides/sql_database_integration.html
  dev_url: https://github.com/alteryx/featuretools-sql
  home: https://pypi.org/project/featuretools-sql/
```

File: recipes/featuretools_sql/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  summary: directly import and convert their relational data into a Featuretools
+  license: BSD-3-Clause
+  license_file: LICENSE
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:42]:
```suggestion
  license_family: BSD
  license_file: LICENSE
```

File: recipes/featuretools_sql/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - connectorx >=0.3.0
+    - snowflake-sqlalchemy >=1.4.2
+
+test:
+  imports:
+    - featuretools_sql
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:36]:
```suggestion
    - pip
  source_files:
    - featuretools_sql/*
```

File: recipes/featuretools_sql/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    - numpy >=1.17.5
+    - pandas >=1.3.0
+    - featuretools >=1.5.0
+    - psycopg2 >=2.9.3
+    - connectorx >=0.3.0
+    - snowflake-sqlalchemy >=1.4.2
+
+test:
+  imports:
+    - featuretools_sql
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:32]:
```suggestion
    - featuretools_sql
    - pytest --pyargs featuretools_sql
```

File: recipes/featuretools_sql/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: 46d090103f7721958a940a10b1a7a40a385278b802134f9ca04ed90d46a45291
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6,<4.0
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:19]:
```suggestion
    - python >=3.8.*
```

File: recipes/featuretools_sql/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,57 @@
+{% set name = "featuretools_sql" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/featuretools_sql-{{ version }}.tar.gz
+  sha256: 46d090103f7721958a940a10b1a7a40a385278b802134f9ca04ed90d46a45291
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:10]:
```suggestion
  sha256: ee99c888f4c205d22e52f69d99bd54ce52366bd3c035586069e60efe77374ae6
```

File: recipes/featuretools_sql/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+  host:
+    - python >=3.8.*
+    - pip
+  run:
+    - python >=3.6,<4.0
+    - numpy >=1.17.5
+    - pandas >=1.3.0
+    - featuretools >=1.5.0
+    - psycopg2 >=2.9.3
+    - connectorx >=0.3.0
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:27]: remove or change?

Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:27]:
```suggestion
    - sqlalchemy >= 1.4.2
    - PyMySQL >= 1.0.2
```

File: recipes/featuretools_sql/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8.*
+    - pip
+  run:
+    - python >=3.6,<4.0
```
Comment (gsheni) [inline recipes/featuretools_sql/meta.yaml:22]:
```suggestion
    - python >=3.8,<4.0
```

File: recipes/featuretools_sql/meta.yaml (hunk 9 above)
Comment (sbadithe) [reply recipes/featuretools_sql/meta.yaml:27]: Yep, updating now. `connectorX` will be removed, replaced by `sqlalchemy` and `pymysql`

File: recipes/featuretools_sql/meta.yaml (hunk 6 above)
Comment (sbadithe) [reply recipes/featuretools_sql/meta.yaml:33]: these would require the user to have set up `mysql` and `postgres` services

File: recipes/featuretools_sql/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+  dev_url: https://github.com/alteryx/featuretools-sql
+  home: https://pypi.org/project/featuretools-sql/
+  summary: directly import your relational data into Featuretools to run automated feature engineering
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - stefaniesmith
```
Comment (xhochy) [inline recipes/featuretools_sql/meta.yaml:50]: The users here need to acknowledge that they like to be maintainers.

Comment (gsheni) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (rwedge) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (dvreed77) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (stefaniesmith) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (thehomebrewnerd) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (machineAYX) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (ozzieD) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (sbadithe) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (tamargrey) [reply recipes/featuretools_sql/meta.yaml:50]: I would like to be a maintainer.

Comment (gsheni) [reply recipes/featuretools_sql/meta.yaml:50]: @xhochy All maintainers have commented.

**Pull request merged**